src/formpack/schema/fields.py

```diff
@@ -688,7 +696,8 @@ def _get_option_label(self, lang=UNSPECIFIED_TRANSLATION, group_sep='/',
         label = self._get_label(lang, group_sep, hierarchy_in_labels)
         option_label = option['labels'].get(lang) or option['name']
         group_sep = group_sep or "/"
-        return label + group_sep + option_label
+
+        return "{}{}{}".format(label, group_sep, option_label)
```

Comment:
Was this to work around `label`, `group_sep`, or `option_label` being `None`? If so, I think we should either revert to the `+` operator _or_ add an assertion—but not silently pass a `None` along to the user. None of these variables should be `None` at this point; if there's no label assigned for the question (`label`) or choice (`option_label`), then the name should be used—and the name cannot be `None`. I think the work you're doing in the other methods should take care of this.

I don't know if we validate `group_sep`: anyone trying to export with `group_sep=None` is shooting themselves in the foot, but if you're worried about it, I'd suggest checking for it and raising an exception in `__init__()` or `export()`.